librz/core/cbin.c

```diff
@@ -5243,3 +5243,22 @@ RZ_API bool rz_core_bin_archs_print(RzBin *bin, RzCmdStateOutput *state) {
 	rz_cmd_state_output_array_end(state);
 	return true;
 }
+
+RZ_API bool rz_core_bin_pdb_load(RZ_NONNULL RzCore *core, RZ_NONNULL const char *filename) {
```

Comment:
Could you please avoid any PDB changes until https://github.com/rizinorg/rizin/pull/1517 is merged?

Reply:
If I don't do this, cutter cannot work.

Reply:
Ok, then. I hope it will not produce any conflicts.

Reply:
Yes, it should not produce conflicts, it seems: https://github.com/rizinorg/rizin/pull/1517/files#diff-3f97fa549a4b7752e60c646c781a7c013401b28b9ddc18efc0a0a5cc3d713d79